airflow/providers/amazon/aws/hooks/redshift_sql.py

```diff
@@ -22,6 +22,8 @@
 from sqlalchemy.engine.url import URL
 
 from airflow.compat.functools import cached_property
+from airflow.models.connection import Connection
```

Comment:
Me use Connection only for annotation so we could specify that it uses only for it

https://github.com/apache/airflow/blob/5802469fbc452a5727c938f033f2753571989d92/airflow/providers/amazon/aws/hooks/base_aws.py#L56-L59

In this module there is small chance that we got circular import error but still better to use it inside `if TYPE_CHECKING` block